src/features/user/constants/data.ts

```diff
@@ -0,0 +1,9 @@
+import  { UserData } from '../types/user-data';
+
+export const DEFAULT_DATA: UserData = {
```

Comment:
czego default data? Musisz doprecyzować w nazwie, nic to nie mówi

Reply:
Już zmienione!